### ISSUE TITLE
Arg parsing bug affecting language pack downloading

### ISSUE THREAD

**Issue by aronasorman** [learningequality/ka-lite#4935]
## Summary

I can't download language packs with a dash in their name, like `es-ES`.

Output:

```
$ kalite manage retrievecontentpack local 'es-ES' '~/src/github.com/fle-internal/content-pack-maker/out/es-ES.zip'
Please only use --option=value or -x123 patterns. No spaces allowed between option and value. The option parser gets confused if you do otherwise.

Will be fixed in a future release.[70307 refs]
[52419 refs]
```

## Branch or installer method

0.16.x

## Traceback or relevant snippet from server.log

```
INSERT TRACEBACK, LOG MESSAGES ETC. HERE
```

## How to reproduce

See log pasted above.

## Screenshots

*If applicable*

## Real-life consequences (anything community should be aware, for instance how it affects your deployment)

For admins who want to install a sublanguage, they'll have to do it through a roundabout way, running `kalite shell` and then `call_command(`retrievecontentpack`...)

**Comment by radinamatic** [comment]
Bummer, since it affects the two largest user groups who would use localized version (es-ES, since we don't have other Latin American Spanish subgroups, and pt-BR), with most of subtitles translated (I think)... :disappointed:

**Comment by aronasorman** [comment]
Yeah, I think fixing this would be important for 0.16. Hence the release blocker tag!

**Comment by radinamatic** [comment]
Apologies, I totally misunderstood that `Will be fixed in a future release.` Thought you were punting it...

**Comment by rtibbles** [comment]
Fixed in #4939
